scipy/stats/_continuous_distns.py

```diff
@@ -5909,19 +5914,33 @@ def _isf(self, q, df):
         return -sc.stdtrit(df, q)
 
     def _stats(self, df):
-        mu = np.where(df > 1, 0.0, np.inf)
-        mu2 = _lazywhere(df > 2, (df,),
-                         lambda df: df / (df-2.0),
-                         np.inf)
-        mu2 = np.where(df <= 1, np.nan, mu2)
-        g1 = np.where(df > 3, 0.0, np.nan)
-        g2 = _lazywhere(df > 4, (df,),
-                        lambda df: 6.0 / (df-4.0),
-                        np.inf)
-        g2 = np.where(df <= 2, np.nan, g2)
+        # 0.0, 1.0, 0.0, 0.0
+        cond = df == np.inf
+        mu = np.where(df > 1 | cond, 0.0, np.inf)
```

Comment:
```suggestion
        mu = np.where(df > 1, 0.0, np.inf)
```
`np.inf > 1`, right?